docs/usage.md

```diff
@@ -159,6 +160,16 @@ Use to skip `FastQC` on obtained reads. This is useful, when the reads are used
 
 Use to specify the fast mode for the `samtools collate` command in the processes `sortExtractMapped`, `sortExtractUnmapped` and `sortExtractSingleEnd`.
 
+### `--reads_in_memory` (optional)
+
+Only relevant in combination with `--samtools_collate_fast`. This is useful for speeding up the processes `sortExtractMapped`, `sortExtractUnmapped` and `sortExtractSingleEnd`.
```

Comment:
I think adding the default value to the docs here would be good too :)

Reply:
Yes, totally agree :)